src/components/AppDetails/AppDetails.js

```diff
@@ -152,26 +220,13 @@ export const AppDetails = ({
                         </em>
                     )}
                 </div>
-                <div>
-                    <ManageInstalledVersion
-                        installedApp={installedApp}
-                        versions={appHubApp?.versions}
-                        onVersionInstall={onVersionInstall}
-                        onUninstall={onUninstall}
-                    />
-                    {installedApp && appHubApp && (
-                        <div>
-                            <h2 className={styles.sectionHeader}>
-                                {i18n.t('Additional information')}
-                            </h2>
-                            <Metadata
-                                installedVersion={installedApp.version}
-                                versions={versions}
-                            />
-                        </div>
-                    )}
-                </div>
-            </section>
+            </section> */}
+            <iframe
+                width="100%"
+                height="1000px"
+                style={{ border: 0 }}
+                src={`http://localhost:8080/static/app/${appHubApp.id}?compatibleWith=${dhisVersion}`}
```

Comment:
this is pointing to AppHub, and passing the DHIS version as a query string to the iframe